src/app/ui-helper.js

```diff
@@ -21,6 +21,13 @@ module.exports = {
       cls);
```

Comment:
The textarea has the benefit that it can be resized and the data made readable. With input it needs to be copy pasted to the outside.